.github/workflows/python-package.yml

```diff
@@ -41,7 +41,7 @@ jobs:
           C_INCLUDE_PATH: /usr/include/cfitsio/
     - name: Test
       run: |
-        source setup_local_crds.sh && pytest -v
+        source setup_local_crds.sh && pytest -v liger_iris_pipeline/tests/test_datamodels.py liger_iris_pipeline/tests/test_dark.py liger_iris_pipeline/tests/test_rop.py liger_iris_pipeline/tests/test_utils.py liger_iris_pipeline/tests/test_normalize.py
         #- name: Build docs
```

Comment:
I am worried we could break some of the older tests if we do not test on CI.
So I suggest to add new tests here when we refactor them.
Once we are done we will set it back to `pytest -v` only.

Reply:
```
liger_iris_pipeline/tests/test_datamodels.py::test_load_iris_image_file PASSED [ 11%]
liger_iris_pipeline/tests/test_dark.py::test_dark_subtraction PASSED     [ 22%]
liger_iris_pipeline/tests/test_dark.py::test_dark_step PASSED            [ 33%]
liger_iris_pipeline/tests/test_rop.py::test_rop1 PASSED                  [ 44%]
liger_iris_pipeline/tests/test_rop.py::test_rop2 PASSED                  [ 55%]
liger_iris_pipeline/tests/test_normalize.py::test_normalize PASSED       [ 66%]
liger_iris_pipeline/tests/test_normalize.py::test_normalize_step PASSED  [ 77%]
liger_iris_pipeline/tests/test_assign_wcs.py::test_assign_wcs_step PASSED [ 88%]
liger_iris_pipeline/tests/test_asdf_schema.py::test_load_schema PASSED   [100%]
```